commit f6c82c73b673ec3c9cce432fe38d5e0076234efd
Author: Chris Wilson <chris@chris-wilson.co.uk>
Date:   Fri Dec 2 10:34:10 2011 +0000

    uxa: Fix runtime linking of previous commit
    
    So much for relying on compiler warnings.
    
    Signed-off-by: Chris Wilson <chris@chris-wilson.co.uk>

diff --git a/src/intel_display.c b/src/intel_display.c
index 95a9488..2183f4d 100644
--- a/src/intel_display.c
+++ b/src/intel_display.c
@@ -409,8 +409,6 @@ intel_crtc_apply(xf86CrtcPtr crtc)
 		}
 	}
 
-	intel_set_gem_max_sizes(scrn);
-
 	if (scrn->pScreen)
 		xf86_reload_cursors(scrn->pScreen);
 
diff --git a/src/intel_driver.c b/src/intel_driver.c
index 451bebd..2828ed6 100644
--- a/src/intel_driver.c
+++ b/src/intel_driver.c
@@ -1170,8 +1170,6 @@ static Bool I830EnterVT(int scrnIndex, int flags)
 			   strerror(errno));
 	}
 
-	intel_set_gem_max_sizes(scrn);
-
 	if (!xf86SetDesiredModes(scrn))
 		return FALSE;
 
